community/bolt/src/test/java/org/neo4j/bolt/v1/runtime/BoltStateMachineTest.java

```diff
@@ -401,4 +411,39 @@ public void actionsDisallowedBeforeInitialized() throws Throwable
         }
     }
 
+    @SuppressWarnings( "unchecked" )
+    @Test
+    public void shouldTerminateOnAuthExpiryDuringREADYRun() throws Throwable
+    {
+        // Given
+        TransactionStateMachine.SPI transactionSPI = mock( TransactionStateMachine.SPI.class );
+        when( transactionSPI.beginTransaction( any() ) ).thenThrow( new AuthExpirationException( "Auth expired!" ) );
```

Comment:
It is a minor Mockito quirk. You have to use `doThrow(...).when(...)` syntax when the method is `void` (then it cannot wrap the return value)